code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -497,7 +497,7 @@ obj/item/asteroid/basilisk_hide/New()
 
 /obj/item/asteroid/goliath_hide/afterattack(atom/target, mob/user, proximity_flag)
 	if(proximity_flag)
-		if(istype(target, /obj/item/clothing/suit/space/rig/mining) || istype(target, /obj/item/clothing/head/helmet/space/rig/mining) || istype(target, /obj/item/clothing/suit/space/plasmaman/miner) || istype(target, /obj/item/clothing/head/helmet/space/plasmaman/miner))
+		if(istype(target, /obj/item/clothing/suit/space/rig/mining) || istype(target, /obj/item/clothing/head/helmet/space/rig/mining) || istype(target, /obj/item/clothing/suit/space/plasmaman/miner) || istype(target, /obj/item/clothing/head/helmet/space/plasmaman/miner) || istype(target, /obj/item/clothing/suit/space/vox/civ/trader) || istype(target, /obj/item/clothing/head/helmet/space/vox/civ/trader))
```

Comment:
We seriously need an is_goli_reinforcable() check or something, this is getting a tad ridiculous